### PR TITLE
fix: wrong cn translation

### PR DESCRIPTION
**Pull request by Decodetalkers**
Issue: https://github.com/linuxdeepin/developer-center/issues/5694 Log:

### CODE REVIEW

**Review by BLumia — APPROVED** [review]

File: translations/dde-control-center_zh_CN.ts [hunk 1]
```diff
@@ -926,7 +926,7 @@
     <name>dccV23::CustomAddAvatarWidget</name>
     <message>
         <source>You have not uploaded a picture, you can click or drag to upload a picture</source>
-        <translation>您还没有上传过头像，可点击或拖搜上传图片</translation>
+        <translation>您还没有上传过头像，可点击或拖曳上传图片</translation>
```
Comment (BLumia) [inline translations/dde-control-center_zh_CN.ts:929]: 刚意识到，拖“拽”写错了吧，

**Review by BLumia — CHANGES_REQUESTED** [review]

File: translations/dde-control-center_zh_CN.ts (hunk 1 above)
Comment (Decodetalkers) [reply translations/dde-control-center_zh_CN.ts]: 哦，我平常用拖曳就

**Review by BLumia — APPROVED** [review]

**Pull request merged**